http_check/datadog_checks/http_check/http_check.py

```diff
@@ -56,14 +56,14 @@ def _check(self, instance):
 
         def send_status_up(logMsg):
             # TODO: A6 log needs bytes and cannot handle unicode
-            self.log.debug(ensure_bytes(logMsg))
```

Comment:
We have to send bytes to `log`, otherwise it will try to encode using `ascii`... Unless there is a way to configure the logger to use `utf-8`.
```Traceback (most recent call last):
  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/datadog_checks/base/checks/network.py", line 30, in check
    statuses = self._check(instance)
  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/datadog_checks/http_check/http_check.py", line 194, in _check
    .format(ensure_unicode(content_match)))
  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/datadog_checks/http_check/http_check.py", line 66, in send_status_down
    self.log.info(loginfo)
  File "/opt/datadog-agent/embedded/lib/python2.7/logging/__init__.py", line 1174, in info
    self._log(INFO, msg, args, **kwargs)
  File "/opt/datadog-agent/embedded/lib/python2.7/logging/__init__.py", line 1293, in _log
    self.handle(record)
  File "/opt/datadog-agent/embedded/lib/python2.7/logging/__init__.py", line 1303, in handle
    self.callHandlers(record)
  File "/opt/datadog-agent/embedded/lib/python2.7/logging/__init__.py", line 1343, in callHandlers
    hdlr.handle(record)
  File "/opt/datadog-agent/embedded/lib/python2.7/logging/__init__.py", line 766, in handle
    self.emit(record)
  File "/opt/datadog-agent/embedded/lib/python2.7/site-packages/datadog_checks/base/log.py", line 21, in emit
    self.format(record)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 8: ordinal not in range(128)```

Reply:
So since you decoded to `utf-8` the url, wherever you log it, if it contains non ascii chars it will fail like the above